### ISSUE TITLE
rss2json

### ISSUE THREAD

**Issue by bluetch** [bluetch/shopee_design#19]
I am using the serve from https://rss2json.com/ to fetch articles for Medium. But recently the rss2json is 500 internal error.

Could I use other npm module to replace it? like https://www.npmjs.com/package/rss2json2

thank you

**Comment by stupergenius** [comment]
Not really. The issue you'll run into (likely) is cross site scripting prevention measures. Services like rss2json.com are designed for this purpose and allow all referers, but sites like medium are not.

One of the better means to accomplishing what you want is to serve this up from whatever hosting service you have. Your server can go fetch the medium RSS feed and convert it to JSON (using one of the various npm libraries to do so if you run a node server) before returning it. Meaning, your react code only needs to change minimally to call your server instead of rss2json.com.

**Comment by bluetch** [comment]
is possible use Medium API to get post via react?

**Comment by bluetch** [comment]
Hi @stupergenius could you teach me how to do it? I am frustration in this case..
